### PR TITLE
[Tizen] Merge the GPU process to the Browser process

### PR DESCRIPTION
**Pull request by pozdnyakov**
Step of the Dual Process model implementation.
See https://lists.crosswalk-project.org/pipermail/crosswalk-dev/2014-December/002308.html for the details.

BUG=XWALK-3203

### CODE REVIEW

**Comment by pozdnyakov** [comment]
To be merged after XWALK-3352 is fixed.

**Comment by rakuco** [comment]
The Tizen Common build failure looks real. Also, can you write a bit more on the commit message and explain why this is helpful and necessary? Otherwise anyone looking at the history will first need to find out what "dual process model implementation" means and how this is related to it.

File: runtime/browser/xwalk_browser_main_parts_tizen.cc [hunk 1]
```diff
@@ -34,6 +34,11 @@ void XWalkBrowserMainPartsTizen::PreMainMessageLoopStart() {
   CommandLine* command_line = CommandLine::ForCurrentProcess();
   command_line->AppendSwitch(switches::kIgnoreGpuBlacklist);
 
+  // switches::kInProcessGPU is not being used here because it
+  // doesn't have the CONTENT_EXPORT macro despite the fact it is exposed by
+  // content_switches.h.
+  command_line->AppendSwitch("in-process-gpu");
```
Comment (hmin) [inline runtime/browser/xwalk_browser_main_parts_tizen.cc]: Bad style here. Please use ```switches::kInProcessGpu``` instead.

Comment (halton) [inline runtime/browser/xwalk_browser_main_parts_tizen.cc]: Nice catch and just say hello to @hmin :)

Comment (rakuco) [inline runtime/browser/xwalk_browser_main_parts_tizen.cc]: There is an explanation in the comment about why it is not being used.

**Comment by rakuco** [comment]
Rendered obsolete by #3299.

**Pull request closed without merging**